### ISSUE TITLE
Not able to find from NPM registry when trying to install.

### ISSUE THREAD

**Issue by prabh08** [PeterStaev/NativeScript-Grid-View#64]
Hello @PeterStaev I am working on the Nativescript 6.1 and using NativeScript-Grid-View but it is not working for me but I read your documentation that "**This repo only supports NativeScript pre-6.0. The latest version of the plugin supporting NS 6+ is availble as part of ProPlugins.**".

When I am trying to install this plugin from https://npm.proplugins.org/-/web/detail/@proplugins/nativescript-grid-view
but it showing this plugin is not available on npm registry. Can you please help me.

Basically I am getting this error **Cannot read property 'widget' of undefined**

**Comment by PeterStaev** [comment]
Hey @prabh08 , ProPlugins is a paid monthly subscription service: https://proplugins.org/subscribe.html . Once you subscribe you can login to the private NPM registry following the instructions [here](https://accounts.nativescript.rocks/subscription/current) and install of the plugins that are on ProPlugins.
